python/lsst/ts/m2gui/controltab/tab_utility_view.py

```diff
@@ -33,7 +39,424 @@ class TabUtilityView(TabDefault):
+            Group.
+        """
+
+        layout = QFormLayout()
+        layout.addRow("Voltage:", self._power_inclinometer["power_voltage_motor"])
+        layout.addRow("Current:", self._power_inclinometer["power_current_motor"])
+
+        return create_group_box("Motor Power", layout)
+
+    def _create_group_power_communication(self):
+        """Create the group of communication power.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QFormLayout()
+        layout.addRow(
+            "Voltage:", self._power_inclinometer["power_voltage_communication"]
+        )
+        layout.addRow(
+            "Current:", self._power_inclinometer["power_current_communication"]
+        )
+
+        return create_group_box("Communication Power", layout)
+
+    def _create_group_inclinometer(self):
+        """Create the group of inclinometer.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QFormLayout()
+        layout.addRow("Inclinometer Angle:", self._power_inclinometer["inclinometer"])
+
+        return create_group_box("Elevation Angle", layout)
+
+    def _create_group_breakers(self, power_type):
+        """Create the group of breakers.
+
+        Parameters
+        ----------
+        power_type : enum `PowerType`
+            Power type.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QVBoxLayout()
+
+        breakers = self.model.utility_monitor.get_breakers(power_type)
+        for breaker in breakers:
+            layout.addWidget(self._breakers[breaker])
+
+        return create_group_box(f"{power_type.name} Breakers", layout)
+
+    def _create_group_temperatures(self, temperature_groups, group_title):
+        """Create the group of temperature sensors.
+
+        Parameters
+        ----------
+        temperature_groups : `list [TemperatureGroup]`
+            List of the temperature groups.
+        group_title : `str`
+            Group title.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QFormLayout()
+        for temperature_group in temperature_groups:
+            sensors = self.model.utility_monitor.get_temperature_sensors(
+                temperature_group
+            )
+
+            for sensor in sensors:
+                layout.addRow(sensor + ":", self._temperatures[sensor])
+
+            self._add_empty_row_to_layout(layout)
+
+        return create_group_box(group_title, layout)
+
+    def _add_empty_row_to_layout(self, layout):
+        """Add the empty row to the layout.
+
+        Parameters
+        ----------
+        layout : `PySide2.QtWidgets.QFormLayout`
+            Layout.
+        """
+        layout.addRow("", create_label())
```

Comment:
Probably a bit better (saves memory of few QLabel and few create_label calls):
```python
layout.addRow(" ", None)
```

Reply:
Fixed in the commit: [4d02847](https://github.com/lsst-ts/ts_m2gui/pull/7/commits/4d028476f7478675d575228b95db083bc1ae6e33). Thanks!